agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaEvent.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
```

Comment:
no need to div as parent since you have v-menu only

Reply:
done